packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -139,14 +141,37 @@ const Alert = ({
       '[Blade: Alert]: SecondaryAction is allowed only when PrimaryAction is defined.',
     );
   }
+  const { theme } = useTheme();
+  const { matchedDeviceType } = useBreakpoint({ breakpoints: theme.breakpoints });
+  const isMobile = matchedDeviceType === 'mobile';
+
   const [isVisible, setIsVisible] = useState(true);
   const contrastType = `${contrast}Contrast` as const;
   const iconSize = isBorderless ? 'large' : 'medium';
   const textSize = isBorderless ? 'medium' : 'small';
 
   const Icon = intentIconMap[intent];
+  let iconOffset: DotNotationSpacingStringToken = 'spacing.1';
+
+  // certain special cases below needs special care for near perfect alignment
```

Comment:
😭

Reply:
I feel uneasy introducing these checks in the code 😅